### ISSUE TITLE
Some specific songs cannot be downloaded.

### ISSUE THREAD

**Issue by xgrasn** [oSumAtrIX/DownOnSpot#61]
Some specific songs are displayed as Unavailable and cannot be downloaded.
They can be played on the official Web Player / official app.

unable to download a track by specifying its URL or SpotifyURI.

Windows 11
Spotify Premium

```
Settings successfully loaded.
Continuing with spotify account: ****
Login succeeded.
Preparing...       | *****

Elapsed second(s): 0
Unavailable!       | *****
Finished download(s) in 1 second(s).
```

**Comment by oSumAtrIX** [comment]
https://github.com/oSumAtrIX/DownOnSpot/issues/53

**Comment by grufkork** [comment]
Should be fixed by #76
